suitability/toolboxes/scripts/NAMDownload.py

```diff
@@ -1,228 +1,170 @@
-# coding: utf-8
 '''
------------------------------------------------------------------------------
-Copyright 2015 Esri
```

Comment:
I'm thinking it was something caught in a diff with an earlier non-functioning version.